packages/react-scripts/config/webpack.config.js

```diff
@@ -568,9 +568,9 @@ module.exports = function(webpackEnv) {
               loader: require.resolve('file-loader'),
               // Exclude `js` files to keep "css" loader working as it injects
               // its runtime that would otherwise be processed through "file" loader.
-              // Also exclude `html` and `json` extensions so they get processed
+              // Also exclude `html`, `wasm` and `json` extensions so they get processed
               // by webpacks internal loaders.
-              exclude: [/\.(js|mjs|jsx|ts|tsx)$/, /\.html$/, /\.json$/],
+              exclude: [/\.(js|mjs|jsx|ts|tsx|wasm)$/, /\.html$/, /\.json$/],
```

Comment:
Interesting! If we were to make this change, perhaps we should add the extension to https://github.com/facebook/create-react-app/blob/b4fe788fe76d90e6e32a1f7b9f61124c2f147545/packages/react-scripts/config/paths.js#L49 

Presumably you could then import `./foo` which could resolves `./foo.wasm`. Webpack appears to support this behavior by default: https://webpack.js.org/configuration/resolve/#resolveextensions

Reply:
Hey! I made and tested the change you suggested.
Also, here's a test repo, for reference: https://github.com/pmrcunha/cra-wasm-test